Library/Formula/wxmac.rb

```diff
@@ -69,6 +71,14 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    if build.with? "static"
+      args.delete("--enable-shared")
```

Comment:
The adding-then-deleting is so that `make` can be run twice with different arguments, once for a shared build, once for a static. I'm very unfamiliar with ruby. Is there a cleaner way to do this?